doc/releases/release-notes-3.7.rst

```diff
@@ -596,6 +585,25 @@ Networking
     maximum supported DUID length.
   * Added documentation page for DHCPv6.
 
+* DNS/mDNS/LLMNR:
+
+  * Fixed an issue where the mDNS Responder did not work when the mDNS Resolver was also enabled.
+    The mDNS Resolver and mDNS Responder can now be used simultaneously.
+  * Reworked LLMNR and mDNS responders, and DNS resolver to use sockets and socket services API.
+  * Added ANY query resource type.
+  * Added support for mDNS to provide records in runtime.
+  * Added support for caching DNS records.
+  * Fixed error codes returned when socket creation fails, and when all results have been returned.
+  * Fixed DNS retransmission timeout calculation.
+
+* gPTP/PTP:
+
+  * Added support for IEEE 1588-2019 PTP.
+  * Added support for SO_TIMESTAMPING socket option to get timestamping information in socket
+    ancillary data.
+  * Fixed race condition on timestamp callback.
```

Comment:
If I get it right this refers to the https://github.com/zephyrproject-rtos/zephyr/commit/c87c36d3130b60451d99ce2dec17ad42788e5d5d. The issue was that `net_pkt` was prematurely unreferenced. And accessing it from timestamp callback might resulted in undefined behaviour - it could be already reused for other packet. I would rephrase it to "Fixed packet reference count upon queuing for TX timestamping, resolving data access issues in timestamp callbacks." but either way LGTM